conans/test/functional/generators/virtualenv_test.py

```diff
@@ -250,11 +248,43 @@ def test_list_variable(self):
             self.ori_path,
             existing_path
         ]))
+
+    @pytest.mark.tool_conan
+    def test_empty_undefined_list_variable(self):
+        self.assertNotIn("WHATEVER", os.environ)
+
+        generator = VirtualEnvGenerator(ConanFileMock())
+        generator.env = {"WHATEVER": ["list", "other"]}
+
+        _, environment = self._run_virtualenv(generator)
+
         # FIXME: extra separator in Windows
         extra_separator = os.pathsep if platform.system() == "Windows" else ""
         self.assertEqual(environment["WHATEVER"],
                          "{}{}{}{}".format("list", os.pathsep, "other", extra_separator))
 
+    @pytest.mark.tool_conan
+    def test_empty_defined_list_variable(self):
+        if os_info.is_windows:
+            # Windows doesn't make distinction between empty and undefined
+            # environment variables.
+            return
```

Comment:
You can use `@pytest.mark.skipif(platform.system() == "Windows", reason="...")`

Reply:
That's what I was looking for! Thanks!